content/docs/api_v2/api_v2_reference.apib

```diff
@@ -3623,6 +3623,47 @@ The user account is identified by the provided authentication token.
     + Attributes
         - data (UserPrivateInfo, required)
 
+
+## Monthly usage [/v2/users/me/usage/monthly{?token,date}]
+
++ Parameters
+
+    + token: `rWLaYmvZeK55uatRrZib4xbZf` (string, required) - API authentication token.
+    + date: `2020-06-14` (string, optional) - Date in the YYYY-MM-DD format.
+
+
+### Get a summary of your monthly usage [GET]
+
+Returns a complete summary of your usage for the current month, both daily and monthly,
+and includes usage of services. It is the same information you will see on your account's
+[Billing page](https://console.apify.com/billing#/usage). The information
+includes your use of storage, data transfer, and request queue usage.
+
+Using the `date` parameter will show your usage in the billing cycle that includes that date.
```

Comment:
Also here instead of `billing cycle` it should be `usage cycle`. Billing cycle is whole subscription period (so can be either month or year) while usage cycle is always monthly.